packages/components/addon/components/hds/avatar/index.hbs

```diff
@@ -0,0 +1,7 @@
+<div class="hds-avatar" ...attributes data-test-avatar aria-hidden="true">
```

Comment:
```suggestion
<div class="hds-avatar" ...attributes aria-hidden="true">
```

Reply:
@alex-ju for our own tests, yes; I left these in because 1. they were in the PDS component and 2. there could be consumers using these data attributes in their own tests. WDYT?

Reply:
Currently only `atlas` would benefit from them; I'm a bit skeptical as we've not done this with other components, but also not too opinionated on this.